core/templates/dev/head/css/oppia.css

```diff
@@ -2135,10 +2137,21 @@ div#ng-curtain {
   font-size: 21px;
   font-weight: 300;
   height: 40px;
-  margin-left: 6px;
   margin-top: 8px;
 }
-.oppia-navbar-profile-dropdown-toggle {
+.oppia-logo-wide {
+  margin-left: 25px;
+}
+.oppia-logo-small {
+  margin-left: 6px;
+}
+
+.nav > li > a.oppia-navbar-tab {
+  font-size: 1em;
+  padding-top: 17px;
+}
+.oppia-navbar-profile-dropdown-toggle,
```

Comment:
Sorry, yep, I don't think that one should be changed. Just the dropdown-related ones.